src/components/shared/transactionAddress/TransactionAddress.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { tokenMap } from '../../../constants/tokens';
+import regex from '../../../utils/regex';
+import transactionTypes, { transactionNames } from '../../../constants/transactionTypes';
+import styles from './transactionAddress.css';
+
+const TransactionAddress = ({
+  address, bookmarks, transactionType, t, token,
+}) => {
+  const account = [...bookmarks.LSK, ...bookmarks.BTC].filter(acc => acc.address === address);
+
+  const renderAddress = () => {
+    if (token === tokenMap.LSK.key) {
+      return account.length ? account[0].title : address;
+    }
+    return account.length ? account[0].title : address.replace(regex.btcAddressTrunk, '$1...$3');
+  };
```

Comment:
Having `return account.length ? account[0].title : ...` twice here doesn't seem DRY.

Could be DRYed with something like this:
```js
const formatter = (token === tokenMap.LSK.key) 
  ? value => value 
  : (value => value.replace(regex.btcAddressTrunk, '$1...$3'));
return account.length ? account[0].title : formatter(address);
```

Reply:
✅